# Generated by Django 3.1.7 on 2022-07-25 04:14

from django.db import migrations, models


class Migration(migrations.Migration):

    initial = True

    dependencies = [
    ]

    operations = [
        migrations.CreateModel(
            name='Error',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('sql', models.TextField(editable=False, null=True)),
                ('exc_type', models.TextField(editable=False, verbose_name='type')),
                ('exc_value', models.TextField(editable=False, verbose_name='value')),
                ('exc_traceback', models.TextField(editable=False, verbose_name='traceback')),
                ('created_at', models.DateTimeField(auto_now_add=True, verbose_name='created')),
            ],
            options={
                'db_table': 'django_execute_sql_error',
                'ordering': ('-created_at',),
            },
        ),
        migrations.CreateModel(
            name='Sql',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('name', models.CharField(max_length=255, unique=True)),
                ('sql', models.TextField()),
                ('success', models.BooleanField(editable=False, null=True)),
                ('updated_at', models.DateTimeField(editable=False, null=True, verbose_name='updated')),
            ],
            options={
                'db_table': 'django_execute_sql_sql',
                'ordering': ('-updated_at',),
            },
        ),
    ]
